src/prompts/template.ts

```diff
@@ -57,22 +57,24 @@ export const templateMissingOptionPrompt = async (options, folderNameAnswers, de
       name: 'template',
       message: 'Please choose which node-mongo API boilerplate template to use',
       choices: templateCollection, //uses full names of templates in prompt for better user experience
-      default: defaultTemplate
+      default: defaultTemplate,
```

Comment:
If you have personal prettier settings that cause these please always disable it when you are working on our repos.

If it's just from your end, then I'd say be careful so that you don't always get sent back like this for these kind of minor changes.

Only the changes you should be making is what I want to see in the file changes. Even adding extra space is not allowed.

Help make those changes - Thanks.

> You can always use the file changes in your editor or the one on your PR to check before requesting for review.